drake/systems/plants/@RigidBodyWing/RigidBodyWing.m

```diff
@@ -658,14 +666,17 @@ function revertEnvVars()
       %   wingvel_world_xzy
       %     velocity of the wing in world coordinates
       %   </pre>
-      
+      options.in_terms_of_qdot = false;
       if (nargout>1)
         
-        [~, J, dJ] = forwardKin(manip,kinsol,kinframe,zeros(3,1));
-        wingvel_world_xyz = J*qd; % assume still air. Air flow over the wing
+        %[~, J, dJ] = forwardKin(manip,kinsol,kinframe,zeros(3,1));
+        [x,J_geometric,dJ_geometric_dq] = forwardKin(manip,kinsol,kinframe,zeros(3,1),options);
```

Comment:
I think I should be blamed for this one. I called it geometric Jacobian in RigidBodyPropellor. m